lib/generators/ruby_llm/chat_ui/templates/views/messages/_message.html.erb.tt

```diff
@@ -4,6 +4,15 @@
     <%%= <%= message_model_name.underscore %>.role&.capitalize %>
   </div>
   <div style="white-space: pre-wrap;"><%%= <%= message_model_name.underscore %>.content %></div>
+  <%% if <%= message_model_name.underscore %>.tool_calls.any? %>
+    <div style="display: flex; flex-direction: column; gap: 3px; align-items: flex-start; font-family: monospace;">
+      <%% <%= message_model_name.underscore %>.tool_calls.each do |tool_call| %>
+        <div style="background: #eee; padding: 5px; border-radius: 4px;">
+          <%%= tool_call.name %>(<%%= tool_call.arguments.to_json %>)
```

Comment:
instead of `to_json` can you find a way to make it look like a ruby method call?